docs/data-formats.md

```diff
@@ -4,7 +4,23 @@ IL linker uses several data formats to control or influence the linking process.
 
 ## Descriptor Format
 
-The `fullname` attribute specifies the fullname of the type in the format specified by ECMA-335. This is in certain cases not the same as the one reported by Type.FullName for example for nested types.
+Descriptors are used to direct linker to always keep some items in the assembly, regardless if linker can find any references to them.
+
+Descriptor XML can be embedded in an assembly. In that case it must be stored as an embedded resource with logical name `ILLink.Descriptors.xml`. To achieve this when build an assembly use this in the project file to include the XML:
```

Comment:
`logical name` sounds a bit offensive to me ;-)

Reply:
That's an MSBuild terminology - I didn't come up with that 😞